### DISCUSSION TITLE
Enhancement: default characters option

### DISCUSSION THREAD

**Discussion by LoafyToast** [FunkinCrew/Funkin#1609] [Ideas]
Perhaps add a setting in the options menu to set the defualt bf, gf, and opponent for anyone who wants to only play as that character for the session, then they wont need to select the characters every time they load a song! sorry if this isn't in the right section i just started using github.

**Comment by TentaRJ** [comment]
Good idea, I'll see about working on something like that, never thought of that before.

Also next time you make a new issue, follow the template.
